src/main/java/org/broadinstitute/consent/http/resources/DaaResource.java

```diff
@@ -194,4 +198,19 @@ public Response deleteDaaForUser(
       return createExceptionResponse(e);
     }
   }
+
+  @POST
+  @PermitAll
+  @Path("/request/{daaId}")
+  public Response sendDaaRequestMessage(
+      @Auth AuthUser authUser,
+      @PathParam("daaId") Integer daaId) {
+    try {
+      User user = userService.findUserByEmail(authUser.getEmail());
+      daaService.sendDaaRequestEmails(user, daaId);
```

Comment:
I think we have some room here to do some additional validation. For example:
1. We might want to know that the user doesn't have an SO to send emails to and return with an error.
2. We might want to check to see if they already have the association - therefore we don't need to send an email
3. I'd love to have a way to prevent spamming here ... probably best to file an additional ticket for that with some potential implementation ideas.